modules/gitops-verifying-argo-cd-self-healing-behavior.adoc

```diff
@@ -0,0 +1,34 @@
+[id="verifying-argo-cd-self-healing-behavior_{context}"]
+= Verifying Argo CD self-healing behavior
+
+Argo CD constantly monitors the state of the deployed applications, detects differences between the specified manifests in Git and live changes in cluster, and automatically corrects them. This behavior is referred to as self-healing. 
```

Comment:
I think there are two "the's" too many and one too little. "Argo CD constantly monitors the state of deployed applications, detects differences between the specified manifests in Git and live changes in the cluster, and automatically corrects them."